static/src/js/util/collections.js

```diff
@@ -220,6 +221,10 @@ export const buildCollectionSearchParams = (params) => {
   // Only include has_granules_or_cwic sort key if the parameter is being used
   if (hasGranulesOrCwic) sortKey.unshift('has_granules_or_cwic')
 
+  if (selectedSortKey[0] !== collectionSortKeys.recentVersion) {
```

Comment:
```suggestion
  // Add a secondary sort-key to CMR queries to ensure we are always sorting collections by their most recent version
  if (selectedSortKey[0] !== collectionSortKeys.recentVersion) {
```
Definitely does not have to be that exact phrasing above